src/test/java/com/snowflake/kafka/connector/ConnectorConfigTest.java

```diff
@@ -16,11 +16,59 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import org.apache.kafka.connect.storage.Converter;
 import org.junit.Assert;
 import org.junit.Test;
 
 public class ConnectorConfigTest {
 
+  public enum CommunityConverterSubset {
+    STRING_CONVERTER(
+        "org.apache.kafka.connect.storage.StringConverter",
+        new org.apache.kafka.connect.storage.StringConverter()),
+    JSON_CONVERTER(
+        "org.apache.kafka.connect.json.JsonConverter",
+        new org.apache.kafka.connect.json.JsonConverter()),
+    AVRO_CONVERTER(
+        "io.confluent.connect.avro.AvroConverter", new io.confluent.connect.avro.AvroConverter());
+
+    private final String name;
+    public final Converter converter;
+
+    CommunityConverterSubset(String name, Converter converter) {
+      this.name = name;
+      this.converter = converter;
+    }
+
+    public String toString() {
+      return this.name;
+    }
+  };
+
+  public enum CustomSfConverter {
```

Comment:
This is fine but we do have
` public static final Set<String> CUSTOM_SNOWFLAKE_CONVERTERS ` which can be resued.

Reply:
i considered using that, but i preferred having the enum